products-1.html

```diff
@@ -33,93 +33,67 @@
 		
 		<div class="wrapper">
 			<div class="products">
-                <h1>All Products</h1>
+                <h1>Skincare Products</h1>
+                <p class="subheading">Products for oily skin</p>
 
                 <div class="container">
+
                     <!-- Each product-box div is a product, but this row specifically
                     has a thicker top box line -->
                     <div class="top-row">
                         <div class="product-box" id="first">
-                            <div class="image"><a href="#"><img src="placeholder.png"></a></div>
-                            <a class="h2" href="#">Product name</a>
-                            <p>Price</p>
+                            <div class="image"><a href="https://www.farmers.co.nz/the-ordinary-niacinamide-10-zinc-1-30ml-6394630"><img src="images/oily/to_niacinamide_large.jpg"></a></div>

```

Comment:
I added all of the product images and prices with clickable external links to purchase the actual product. (This applies to all 4 product pages.)